kolibri/plugins/device_management/assets/src/views/SelectContentPage/ContentTreeViewer.vue

```diff
@@ -29,12 +29,11 @@
         </thead>
         <tbody slot="tbody">
           <ContentNodeRow
-            v-for="node in annotatedChildNodes"
-            v-if="showNode(node)"
+            v-for="node in showableAnnotatedChildNodes"
+            :key="node.id"
```

Comment:
did it complain about having both a `v-for` and a `v-if` on the same element?

Reply:
Oh, I thought we already had that rule.

Reply:
Yeah, it's this rule `no-use-v-if-with-v-for`